src/Cdn/Cdn/Models/Endpoint/PSDeliveryRuleCacheExpirationAction.cs

```diff
@@ -20,6 +20,8 @@ namespace Microsoft.Azure.Commands.Cdn.Models.Endpoint
 {
     public class PSDeliveryRuleCacheExpirationAction: PSDeliveryRuleAction
     {
-        public PSCacheExpirationActionParameters Parameters { get; set; }
```

Comment:
@hytao is this change needed? If this `PSDeliveryRuleCacheExpirationAction` type is exposed to the user in any way (_e.g._, as an output or parameter type), then this will be a breaking change as the `Parameters` property is no longer accessible by the user.

Reply:
It is not necessary. I can revert the change here